### ISSUE TITLE
Resident Evil: Revelations 2 - Some stuff is not rendering

### ISSUE THREAD

**Issue by kassindornelles** [Joshua-Ashton/d9vk#72]
Characters and some objects do not render. :frog:
![image](https://user-images.githubusercontent.com/40970965/57393163-e73ccd00-7198-11e9-884b-210e96a0d250.png)
(character should be there but is not, probably got eaten by a frog)

### Software information
Resident Evil Revelations 2 / Biohazard Revelations 2

### System information
- GPU: GTX 750 2 GB
- Driver: 430.39
- OS: Windows 10 Home 

### Apitrace file(s)
https://drive.google.com/file/d/1Gc62sm8ubgFGibGF4TXUGjIA_sfpswRa/view?usp=sharing

### Log files
https://pastebin.com/UcnvCmjn

**Comment by dlshinobi** [comment]
My guess it's the same issue as Dragon's Dogma Dark Arisen, after this commit / build https://git.froggi.es/joshua/d9vk/commit/26f3dfe0deba5b89d9bc894e745359fc16be4298  all NPC characters disappeared / not rendered. Previous commit / build worked fine. Can you test this d9vk build https://git.froggi.es/joshua/d9vk/-/jobs/368/artifacts/download to confirm?

**Comment by kassindornelles** [comment]
> My guess it's the same issue as Dragon's Dogma Dark Arisen, after this commit / build https://git.froggi.es/joshua/d9vk/commit/26f3dfe0deba5b89d9bc894e745359fc16be4298 all NPC characters disappeared / not rendered. Previous commit / build worked fine. Can you test this d9vk build https://git.froggi.es/joshua/d9vk/-/jobs/368/artifacts/download to confirm?

of course, i can try later.

**Comment by kassindornelles** [comment]
![image](https://user-images.githubusercontent.com/40970965/57404514-81f5d580-71b2-11e9-90a9-16ee345969a6.png)

@dlshinobi yes, is working with that build https://git.froggi.es/joshua/d9vk/-/jobs/368/artifacts/download

**Comment by dlshinobi** [comment]
Hmm, good to know. I hope Joshua would fix this bug somehow :)

**Comment by Joshua-Ashton** [comment]
Fixed by https://github.com/Joshua-Ashton/d9vk/commit/ae67d4c862de73c1fef7b6506abe7b99d530787e

**Comment by SergeyLatyshev** [comment]
For me (Ubuntu 18.04) RE:R2 shows nothing but the loading screen instead of 3D rendering, but older DX9 RE games have this problem too (RE1, RE0, RE: Revelations 1). RE4 just crashes or stucks at loading.

**Comment by kassindornelles** [comment]
@SergeyLatyshev sorry, i'm kinda late... but i tested RE4 HD Edition with D9VK and it works just fine
maybe something wrong on your setup?
are you trying to run the old ubisoft port for PC?

**Comment by SergeyLatyshev** [comment]
It's the Steam version.

I tried d9vk-0.11 today, and it works! I think it didn't before because I was using 32-bit wineprefix (RE 1 and 0 need 32-bit) and I'm not sure if I have tried the default 64-bit wineprefix with d9vk for RE4. Anyway, it works now, and I have finished chapter 1-1 without significant problems except for the game crashes when I change anti-aliasing settings, and sound is out of sync in cutscenes.

But Resident Evil: Revelations 2 still does not work for me :(
